### PR TITLE
Add route for editing comments

### PR DESCRIPTION
**Pull request by wheelsandcogs**

### CODE REVIEW

File: pages/project-version/middleware/index.js [hunk 1]
```diff
@@ -10,34 +10,37 @@ const getVersion = () => (req, res, next) => {
     .catch(next);
 };
 
+const extractComments = task => {
```
Comment (lennym) [inline pages/project-version/middleware/index.js]: Minor beef - this is in a middleware module and is not a middleware.

Comment (lennym) [reply pages/project-version/middleware/index.js]: I'd create `./pages/project-version/lib/extract-comments.js` and put this function there.

Comment (wheelsandcogs) [reply pages/project-version/middleware/index.js]: done

File: pages/project-version/lib/extract-comments.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+      return {
+        id,
+        field: activity.event.meta.payload.meta.field,
+        comment,
+        deleted,
+        // we want to show the date of the following status change, not the comment submission.
+        createdAt: ([...statusChanges].reverse().find(s => s.createdAt > createdAt) || {}).createdAt,
+        author: `${firstName} ${lastName}`,
+        isNew
+      };
```
Comment (lennym) [inline pages/project-version/lib/extract-comments.js]: I know this is code that you've just moved from one place to another, but this destructuring to then immediately restructure is kinda wild.

I'm not going to beef for the purposes of merging this though.

**Pull request merged**